mailgun/route_handlers.py

```diff
@@ -64,11 +84,7 @@ def handle_mailing_list_email_route(request):
                                u"for %s, but we've already handled that.  "
                                u'Skipping.', recipient, message_id)
                 continue
-
-        try:
-            _handle_recipient(request, recipient)
-        except JsonResponse as error_response:
```

Comment:
Not following you here -- catching `JsonResponse` just returned the response, so where would the error content be logged if you caught it here and re-`raise`d it? Can you recommend an approach?